### PR TITLE
Fix/create tasks bug

### PR DESCRIPTION
**Pull request by guiwustro**
# Description
This PR is related to the 3 Medium BUGS in the HGN Phase bugs. You can check more details about the bug at the [DOCS ](https://docs.google.com/document/d/1n5NIb3dbiSrIRuOG1ZCwt34pEP1poqddCv1guaQml88/edit), but this PR will change:
- The default option when creating a new task (now is Housing); 
CreatingANewTaskOptionDefault
- If a task has a MOTHER, It means, if a new task is being creating as SUB-TASK (above a task), the default classification will be the mother (but It can be change using the select);
- Correct the time registered by a volunteer in the correct classification;
-
## How to test:
-  Login as admin -> go to OtherLinks -> Projects -> WBS -> Add task 
- After you should add a new task and assign to a volunteer account that you have, and in the volunteer account, you should check if the time is being to the correct classification. (I made a video testing this just to show how I tested, It can be helpful also).

### CODE REVIEW

**Comment by LeeSingChanYau** [comment]
Hey Guilherme I tested it out and everythings works just like in the video.

File: src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx [hunk 1]
```diff
@@ -91,7 +91,7 @@ const AddTaskModal = (props) => {
     {value:"Other",label:"Other"}
   ]
   const [projectCategory, setProjectCategory] = useState('');
-  const [classification, setClassification] = useState('');
+  const [classification, setClassification] = useState('Housing');
```
Comment (LeeSingChanYau) [inline src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx:94]: Defaults to housing.

**Review by LeeSingChanYau — DISMISSED** [review]
Looks good!

File: src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx [hunk 2]
```diff
@@ -91,7 +91,7 @@ const AddTaskModal = (props) => {
     {value:"Other",label:"Other"}
   ]
   const [projectCategory, setProjectCategory] = useState('');
```
Comment (navyasrim) [inline src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx:93]: are we using this anywhere , if not you can remove them..

**Comment by guiwustro** [comment]
Updated! Please, someone can check it again?  The only thing changed was the default option added to the project selected.
Thank you!

File: src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx (hunk 2 above)
Comment (navyasrim) [reply src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx]: removed ..works as expected.. didn't test hours remaining two points working as expected.

**Review by navyasrim — COMMENTED** [review]
line 635,636 seems to be duplicated, we cannot have multiple attributes with same name, please address.

**Comment by navyasrim** [comment]
> > line 635,636 seems to be duplicated, we cannot have multiple attributes with same name, please address.
> 
> Its not related to my PR , I think the duplicated classname atributte is in another files also.
I just reviewed the hole page and left the comment..as the functionality is working as expected, i am ignoring the comments and approving.

**Review by navyasrim — APPROVED** [review]
looks good to me in the lines of code changes..

**Pull request merged**